community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -126,11 +125,6 @@ static HttpConnector httpConnector( String key )
             "" + Math.min( Runtime.getRuntime().availableProcessors(), 500 ),
             range( 1, JettyThreadCalculator.MAX_THREADS ) );
 
-    @Description("If execution time limiting is enabled in the database, this configures the maximum request execution time.")
-    @Internal
-    Setting<Long> webserver_limit_execution_time =
```

Comment:
Since it's unsupported I recon we don't need to have migration / info / warning about it being replaced, do we?